pkg/controller/kappnav/kappnav_controller.go

```diff
@@ -223,6 +229,24 @@ func (r *ReconcileKappnav) Reconcile(request reconcile.Request) (reconcile.Resul
 		},
 	}
 
+	// The kappnav application
+	kappnavCR := &appv1beta1.Application{
+		ObjectMeta: *kappnavName,
+	}
+	kappnavCRAnnotations := map[string]string{
+		"kappnav.application.hidden": "true",
+	}
+
+	// Create or update the kappnav Application
+	err = r.CreateOrUpdate(kappnavCR, instance, func() error {
+		kappnavutils.CustomizeApplication(kappnavCR, instance, kappnavCRAnnotations)
```

Comment:
Did you intend for this Application to be empty except for metadata?

Reply:
Yeah, since we don't display it, there's no value to having component kinds.